src/lib/classes/QContext.svelte.ts

```diff
@@ -57,6 +57,6 @@ export default class QContext<T> {
    * Inner value of the context
    */
   get value() {
-    return this.#state;
+    return this.#state as T;
```

Comment:
Could you add that as return type of the getter instead?

Reply:
I can't, typescript doesn't want to me to assign `T | undefined` to `T` (the `undefined` comes from the `$state` definition at the top and should be there)